pkg/console/controllers/downloadsdeployment/controller.go

```diff
@@ -81,13 +78,13 @@ func NewDownloadsDeploymentSyncController(
 }
 
 func (c *DownloadsDeploymentSyncController) Sync(ctx context.Context, controllerContext factory.SyncContext) error {
-	operatorConfig, err := c.operatorConfigClient.Get(ctx, api.ConfigResourceName, metav1.GetOptions{})
+	operatorConfig, err := c.consoleOperatorLister.Get(api.ConfigResourceName)
 	if err != nil {
 		return err
 	}
-	updatedOperatorConfig := operatorConfig.DeepCopy()
+	operatorConfigCopy := operatorConfig.DeepCopy()
```

Comment:
Not new, but why do we even need a copy? Do we make changes to this resource somewhere?

Reply:
You actually wouldn't need a copy in this function, but you might need it further on I believe.

When you're using listers, a `Get()` method works with an informer cache. It actually returns a pointer *into* the cache. If you did not create a copy, you would be mutating your local cache for the object, which may have unwanted consequences and might prevent you from being able to update the object in the cluster (until the cache is refreshed again).

Reply:
Yeah. From what I can tell we never actually muatate this resource, so it seems like we could get rid of the deep copy operation altogether. But, we don't need to address it here, it just caught my eye.